src/models/media-container/search-result/directory/library-section-id.yaml

```
@@ -0,0 +1,6 @@
+type: object
+properties:
+  librarySectionID:
+    description: "The unique identifier of the library section containing this item."
+    type: integer
+    example: 3 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)